.github/workflows/create-or-update-release-pr.yml

```diff
@@ -0,0 +1,61 @@
+name: Create or Update Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: Release type
+        options:
+          - patch
+          - minor
+      release_scope:
+        type: choice
+        description: Release Scope
+        options:
+          - experimental
+          - sdk
+          - all
+
+jobs:
+  create-or-update-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fork
+        run: gh repo fork open-telemetry/opentelemetry-js
```

Comment:
Do we only need to do this fork *once*, and then change this workflow? I would have thought the forking would be a one-time thing.

Reply:
We can just run the forking step every time we run the workflow, as it's just a single API call. Since the `gh` binary is already available, it does only add about a second to the workflow run. If the fork already exists, then this command will exit with a success code, logging that the fork already exists, effectively becoming a no-op. :slightly_smiling_face:

Here's an example where I tested that: https://github.com/pichlermarc-sample-organization/test-release-pr/actions/runs/10147085271/job/28056647373 :slightly_smiling_face: